# srv file for service to get the details of assembly shipments

---
bool assembly_shipment_present
string order_id
AssemblyShipment[] assembly_shipments